include/nbl/asset/utils/CHLSLCompiler.h

```diff
@@ -8,6 +8,9 @@
 #include "nbl/asset/utils/ISPIRVOptimizer.h"
 #include "nbl/asset/utils/IShaderCompiler.h"
 
+#include <combaseapi.h>
+#include <dxc/dxc/include/dxc/dxcapi.h>
```

Comment:
don't spill this stuff into headers, keep anything that uses these headers in the cpp file

Reply:
then he needs to forward declare some IDxc stuff because CHLSLCompiler holds a pointer to them